localstack/services/sqs/sqs_listener.py

```diff
@@ -24,6 +25,32 @@ def forward_request(self, method, path, data, headers):
                 encoded_data = urlencode(req_data, doseq=True)
                 request = Request(data=encoded_data, headers=headers, method=method)
                 return request
+            elif req_data.get('Action', [None])[0] == 'SendMessage':
+                queue_url = req_data.get('QueueUrl', [None])[0]
+                queue_name = queue_url[queue_url.rindex('/') + 1:]
+                message_body = req_data.get('MessageBody', [None])[0]
+                if lambda_api.process_sqs_message(message_body, queue_name):
+                    # If an lambda was listening, do not add the message to the queue
+                    new_response = Response()
+                    new_response._content = (
+                        '<?xml version="1.0"?>'
+                        '<SendMessageResponse xmlns="http://queue.amazonaws.com/doc/2012-11-05/">'
```

Comment:
nit: Instead of inlining the entire XML string here, can we define a root level constant at the top of the file? Also, we can reuse the `xmlns` namespace URL.